alpine-docker/universal.Dockerfile

```
@@ -0,0 +1,92 @@
+GCC_VER = ${GCC_VERSION}
+MUSL_VER = ${MUSL_VERSION}
+GMP_VER = ${GMP_VERSION}
+MPC_VER = ${MPC_VERSION}
+MPFR_VER = ${MPFR_VERSION}
+LINUX_VER = ${LINUX_VERSION}
+ISL_VER = ${ISL_VERSION}
+EOF
+
+ARG TARGET_MAK_FILE
+
+ADD config-base.mak ${TARGET_MAK_FILE} .
```

Hadolint (2.12.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)